src/components/extensive/WizardForm/index.tsx

```diff
@@ -162,45 +160,64 @@ function WizardForm(props: WizardFormProps) {
     done: props.tabOptions?.markDone && index < selectedStepIndex,
     disabled: props.tabOptions?.disableFutureTabs && index > selectedStepIndex,
     body: (
-      <FormStep
-        id="step"
-        formHook={formHook}
-        fields={step.fields}
-        title={step.title}
-        subtitle={step.subtitle}
-        onChange={onChange}
-        className="h-[calc(100vh-287px)] overflow-auto"
-      >
-        <FormFooter
-          className="mt-12"
-          backButtonProps={
-            !props.hideBackButton
-              ? {
-                  children: props.backButtonText || t("Back"),
-                  onClick: () => {
-                    if (selectedStepIndex > 0) {
-                      setSelectedStepIndex(n => n - 1);
-                    } else {
-                      props.onBackFirstStep();
+      <div className="h-[calc(100vh-287px)] overflow-auto">
+        {index === 0 && step.title === "Site Overview" && (
+          <div className="w-full bg-white px-16 pt-8">
+            <div className="flex gap-4 rounded-lg bg-tertiary-80 p-6">
+              <Text variant="text-16-bold" className="text-white">
+                {t(`Note: Project polygons are editable through a new geometry-focused workflow that is accessible by
+                clicking on this link.`)}
+              </Text>
+              <Button
+                variant="text"
+                className="text-14-bold nowrap whitespace-nowrap rounded-lg border-2 border-transparent bg-[#ffb88891] px-4 py-0 uppercase text-white hover:border-white"
+              >
+                edit polygon
+              </Button>
+            </div>
+          </div>
+        )}
+        <FormStep
+          id="step"
+          formHook={formHook}
+          fields={step.fields}
+          title={step.title}
+          subtitle={step.subtitle}
+          onChange={onChange}
+        >
+          <FormFooter
+            className="mt-12"
+            backButtonProps={
+              !props.hideBackButton
+                ? {
+                    children: props.backButtonText || t("Back"),
+                    onClick: () => {
+                      if (selectedStepIndex > 0) {
+                        setSelectedStepIndex(n => n - 1);
+                      } else {
+                        props.onBackFirstStep();
+                      }
                     }
                   }
-                }
-              : undefined
-          }
-          submitButtonProps={{
-            children:
-              selectedStepIndex < lastIndex
-                ? props.nextButtonText || t("Save and continue")
-                : props.submitButtonText || t("Submit"),
-            onClick: formHook.handleSubmit(onSubmitStep),
-            disabled: (selectedStepIndex === lastIndex && props.submitButtonDisable) || formHasError
-          }}
-        />
-      </FormStep>
+                : undefined
+            }
```

Comment:
if you are going to be using a conditional in the following way:
```
property={someCondition ? component : undefined}
```

there is no point in the undefined, please rewrite it as:
```
property={someCondition && component}
```

Reply:
sorry, but backButtonProps acept undefined and Type 'false' has no properties in common with type 'Omit<IButtonProps, "color" | "variant">'.